core/frontend/render-modes/render-mode-interface.php

```diff
@@ -0,0 +1,50 @@
+<?php
```

Comment:
Why do you need an interface if there is already an abstract class?

Reply:
I order to validate the class type without create an instance.
https://github.com/elementor/elementor/pull/12010/files#diff-8e13e40fed5c012a5e93399f9f22a20dR60-R63

Reply:
The problem with interfaces in PHP is that you have to declare all your methods as public, even though they should be protected or private. Additionally, forcing interface implementation is uncommon in our development conventions and needs further discussion.